.github/workflows/docs-and-linting.yml

```diff
@@ -9,8 +9,9 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        # current Go releases plus the version in the go.mod are tested
-        go: ['go.mod', 'oldstable', 'stable']
+        # Current Go releases plus the version in go.mod and schema/go.mod are tested.
+        # See also BUILD_SPEC_MODULE_ONLY below.
+        go: ['1.18', '1.21', 'oldstable', 'stable']
```

Comment:
The `go.mod` value here was actually really intentional to source the value directly from `go.mod` (see the conditionals below in the `setup-go` action usage).

Are there very many reasons for us to test the version in the top-level `go.mod`, or should we just use `schema/go.mod` for the "go.mod version" case and call it a day?  I think `identity/chainid_test.go` is the only `_test` file outside `schema`, so I don't think we're getting much extra coverage by testing on 1.18 explicitly (except added complexity in the `Makefile`).